elements/rh-spinner/docs/10-style.md

```diff
@@ -9,9 +9,12 @@ A spinner is an animated line segment that follows a track and may include an op
           alt=" Anatomy of a spinner with annotations; number 1 is pointing to the track, number 2 is pointing to the indicator, and number 3 is pointing to the optional text label",
           src="../spinner-anatomy.png" %}
 
+<br>
```

Comment:
shouldn't need this, but might need to add a single newline inbetween the last list item and the `{.example-notes}`

Reply:
see https://www.npmjs.com/package/markdown-it-attrs#ambiguity